src/understanding-workflow/components.md

```diff
@@ -76,8 +76,8 @@ processes.
 **Project Location:** [deis/minio](https://github.com/deis/minio)
 
 All of the Workflow components ship their persistent data to cluster configured
-S3 compatibile Object Storage. For example, database ships its WAL files,
-registry stores Docker images, and slugbuilder stores slugs.
+Object Storage. For example, database ships its WAL files, registry stores Docker images,
```

Comment:
I don't think this sentence is clear: "All of the Workflow components ship their persistent data to cluster configured Object Storage."

Here's a proposed rewrite: "All of the Workflow components that need to persist data will ship them to the object storage that was configured for the cluster."